pettingzoo/classic/chess/chess_utils.py

```diff
@@ -8,8 +8,8 @@ def boards_to_ndarray(boards):
     bits = np.unpackbits(arr8)
     floats = bits.astype(bool)
     boardstack = floats.reshape([len(boards), 8, 8])
-    boardimage = np.transpose(boardstack, [1, 2, 0])
-    return boardimage
+    boardstack = np.flip(np.transpose(boardstack, [1, 2, 0]), axis=[0, 1])
```

Comment:
Can you leave a comment above this line or in the docstring or something explaining why you're calling np.flip? I'm assuming it's the issue you noted in discord with the left side of the board being displayed as the right side for some reason, but future devs would likely be confused by this and want some explanation. Ideally we'd get to the bottom of it and see why it's doing that mirroring in the first place but I'm fine just fixing it for now and noting that there's some other underlying problem that leads to things being mirrored

Reply:
Also not 100% sure if boardimage is the right variable name but I say probably just keep that name for consistency's sake, to show which part is stacking and which is turning it into the actual image

Reply:
I add a comment of why I use `np.flip`. I also add more explanation to this update in the pull request. 
To summarize, in the last version, the function `boards_to_ndarray` outputs an observation that is rotated by 180 degrees compared to `env.render()`.

Reply:
The naming is my typo. Now it is back to `boardimage`.

Reply:
What underlying condition caused the previous version to flip it though? I would think we could figure out the math somehow and resolve it. But maybe it’s due to Pygame’s rendering flipping things in observe? Although the underlying chess board I imagine you can do some sort of function to get it as an array. Maybe we can look into using that or at least check to see how it works.

Reply:
The prevision 180 rotation is contributed by two factors. The line 8 `bits = np.unpackbits(arr8)` gives a bit map, by default, increments from RHS to LHS. Another factor is that the env.render() shows the first line at the bottom but `numpy` shows the first line on the top.

I am going to make comments and optimization about these in next revision.